dbt_subprojects/solana/models/_sector/dex/bot_trades/solana/platforms/bonkbot/schema.yml

```diff
@@ -47,6 +47,7 @@ models:
             - block_month
             - tx_id
             - token_address
+            - fee_receiver
```

Comment:
this needs added to model config block. this is only a test after the models run. adding this field to model config could def fix the dupes, if that was an issue.